### PR TITLE
replace 7.1.9 with 7.2

### PR DESCRIPTION
**Pull request by demonking**
After uncomment nginx config, i was wondering why it was not working out of the box.

### CODE REVIEW

**Comment by schmunk42** [comment]
How's nginx config related to the PHP version?
Shouldn't we go with 5.3 now?

**Comment by demonking** [comment]
Your default Version for apache php is:

PHP_BASE_IMAGE_VERSION=7.2-apache

**Comment by schmunk42** [comment]
Sorry :see_no_evil: I meant `7.3` for both.

**Comment by demonking** [comment]
I do not know what speaks against it.
Should i test and make a  new pull request?

**Comment by schmunk42** [comment]
Yes, please.

File: .env-dist [hunk 1]
```diff
@@ -16,7 +16,7 @@ PHP_ENABLE_XDEBUG=0
 # COMPOSE_PROJECT_NAME=yii-fpm
 # COMPOSE_FILE=docker-compose.yml:docker-compose.fpm-nginx.yml
 # DOCKERFILE_FLAVOUR=debian
-# PHP_BASE_IMAGE_VERSION=7.1.9-fpm
+# PHP_BASE_IMAGE_VERSION=7.2-fpm
```
Comment (samdark) [inline .env-dist:19]: Should be 7.4 now?

**Comment by samdark** [comment]
https://github.com/yiisoft/yii-docker/pull/6 should solve the issue.

**Pull request closed without merging**